Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/physics/PhysicObject.cpp

```diff
@@ -0,0 +1,152 @@
+#include "PhysicObject.hpp"
+#include "../architecture/util.hpp"
```

Comment:
put the include files inside hpp file

Reply:
Some are usefull only in the cpp (using of algo). Are you sure to put whole of them into the hpp ?